### PR TITLE
[AR-30] Mission and Vision Component

### PR DESCRIPTION
**Pull request by alex-baltz-17**
image

### CODE REVIEW

File: app/_components/Content/Header/components/Heading/index.tsx [hunk 1]
```diff
@@ -1,14 +1,17 @@
-import clsx from "classnames";
-import styles from "./styles.module.scss";
+import { cn } from "lib/utils";
 
 const Heading = (props: HeadingProps) => {
   return (
-    <h3 className={clsx(props.styles, styles["wrapper"])}>{props?.label}</h3>
+    <h3 className={cn("w-full text-xxl", props.styles)}>
+      {props?.label}
+      {props.icon && <img src={props.icon} alt="icon" />}
```
Comment (idiglove) [inline app/_components/Content/Header/components/Heading/index.tsx:7]: 👍🏽

File: app/_components/Content/HeaderInfoWithTabs/index.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  const tabs = ["Community", "Who we are", "Our Team", "Join Us"];
+  const [activeTab, setActiveTab] = useState<String>(tabs[0]);
+
+  return (
+    <>
+      <Navbar />
+      <div className={cn(`text-left text-white pb-4 px-12 bg-dark pt-12`)}>
+        <h1
+          className={cn(`text-xxl5 font-Rubik-Regular leading-80`)}
+          style={{ lineHeight: "80px" }}
```
Comment (idiglove) [inline app/_components/Content/HeaderInfoWithTabs/index.tsx]: i think u can do a tailwind arbitrary?
try leading-[80px]

File: app/_components/Content/HeaderInfoWithTabs/index.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        >
+          We’re a small team passionate about building community through
+          boardgames.
+        </h1>
+        <p
+          className={cn(`font-Quicksand-Light text-xl inline-block`)}
+          style={{ lineHeight: "80px" }}
```
Comment (idiglove) [inline app/_components/Content/HeaderInfoWithTabs/index.tsx]: same here

File: tailwind.config.js [hunk 4]
```diff
@@ -101,9 +102,7 @@ module.exports = {
         "Rubik-Regular": "Rubik-Regular",
         "Rubik-Light": "Rubik-Light",
         "Rubik-Bold": "Rubik-Bold",
-      },
-      width: {
-        90: "90%",
```
Comment (idiglove) [inline tailwind.config.js:106]: why did we remove width.90?

**Review by idiglove — APPROVED** [review]
Just need some changes, and this is ok on my side
We will merge it to feat/AR-29 :)

File: tailwind.config.js (hunk 4 above)
Comment (alex-baltz-17) [reply tailwind.config.js:106]: I added it for feat/AR-29 but I saw that I wasn't actually using it, so I removed it here.

**Review by idiglove — APPROVED** [review]
looks good to me!

**Pull request merged**